### PR TITLE
feat: update member page

### PR DESCRIPTION
**Pull request by raaynaldo**
## Linked Issue

#995 

## Description

Update flare, badges, and location on my member page.

CleanShot 2023-10-09 at 16 27 23

PS: I joined last year hacktoberfest last year, but I wasn't part of VC

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

**Comment by raaynaldo** [comment]
Hi @danieltott, I just realized I think we should update the `_EXAMPLE.ts` to add the location placeholder, so new member could get the location?

or maybe I should create an issue (haven't created any issue yet).

**Review by danieltott — CHANGES_REQUESTED** [review]
Thanks @raaynaldo - one quick update requested here.

Good call on the _Example - I'll get that updated.

File: members/members/raaynaldo.ts [hunk 1]
```diff
@@ -46,5 +46,10 @@ export const raaynaldo: MemberObject = {
 			title: 'Leetcode',
 		},
 	],
-	badges: [],
+	badges: ['Hacktoberfest2022', 'Hacktoberfest2023'],
```
[the remark this answers is not in the document]
Comment (raaynaldo) [reply members/members/raaynaldo.ts]: My bad. Thanks for the comment.

**Comment by raaynaldo** [comment]
@danieltott You're welcome. I can update the example if it's okay. I'm curious how to start an issue as well tbh.

**Review by danieltott — APPROVED** [review]
👍🏼

**Pull request merged**